tests/integration/llm/prepare.py

```diff
@@ -350,6 +350,34 @@
     }
 }
 
+lmi_dist_model_list = {
+    "gpt-neox-20b": {
+        "option.model_id": "EleutherAI/gpt-neox-20b",
+        "option.tensor_parallel_degree": 4,
+        "option.max_rolling_batch_size": 4,
+        "load_on_devices": 0
```

Comment:
we dont need to add load_on_devices now. We can also remove the entryPoint.

Reply:
Changed.